### ISSUE TITLE
Uri and fluent interface

### ISSUE THREAD

**Issue by uksus70** [chriseldredge/Lucene.Net.Linq#38]
Seems like `Uri` type is not supported in fluent configuring. Previously I had attribute mapping:

```
        [Field("Url")]
        public IEnumerable<Uri> Urls { get; set; }
```

And now I'm using fluent configration:

```
map.Property(x => x.Urls).ToField("Url");
```

This property does not appear in the index.
Version 3.2.53

**Comment by uksus70** [comment]
I have figured out that specifying type converter solves the problem:

```
map.Property(x => x.Urls).ToField("Url").ConvertWith(new UriTypeConverter());
```

However it was not necessary with attribute mapping.

**Comment by chriseldredge** [comment]
Fixed in 2d35eda7b1fc4754740ce83fd5e63ac0e05900d8.

**Comment by chriseldredge** [comment]
Released: https://nuget.org/packages/Lucene.Net.Linq/3.2.54
